Makefile

```diff
@@ -1,3 +1,5 @@
+include Toolchain.mk
```

Comment:
Having the `include` at the top makes `$(BIN):` the first and thereby default target changing it from the `all:` when running `make` without a target

It retains the previous behaviour if the `include` is moved right after the `all:` target

Reply:
Oh, I've never used default target, so wasn't even aware it is available :)

I think `.DEFAULT_GOAL := all` is a cleaner approach as it's explicit (hard to overlook later on) and allows us to keep vars above all targets.

Fixed, thanks! :)